src/SignalR/server/Core/src/HubConnectionContext.cs

```diff
@@ -583,6 +583,9 @@ private static void AbortConnection(object state)
             finally
             {
                 _ = InnerAbortConnection(connection);
+
+                // Use _streamTracker to avoid lazy init from StreamTracker getter if it doesn't exist
+                connection._streamTracker?.CompleteAll();
```

Comment:
No exception passed ever?

Reply:
That is an open question, thoughts?

Reply:
I think we should always complete the stream with some sort of "The underlying connection was aborted." exception when the stream doesn't end gracefully due to an end stream message.